tests/main/snap-set/task.yaml

```diff
@@ -0,0 +1,76 @@
+summary: Check that `snap set` runs apply-config hook.
+
+systems: [-ubuntu-core-16-64]
```

Comment:
Yeah, the repack seems to have problems. I copied that line from the other tests that repack (snap-run-hook and snapctl), but it would probably be better to figure out a way to make these tests run on ubuntu-core :stuck_out_tongue: . I'll take a look at that this morning.